### ISSUE TITLE
Engine configuration dump

### ISSUE THREAD

**Issue by koen-serry** [trimou/trimou#133]
Would it make sense to dunp the engine configuration only  if "org.trimou.engine.config.debugMode" is true. Instead of all the time?

This would then be the same behaviour as logback/slf4j which also dumps its configuration if the debug is set

**Comment by mkouba** [comment]
Hi @koen-serry, the engine config is logged with the `DEBUG` level so it really depends on the logging config - if you have the `INFO` level set (which should be the default for production) the message is just ignored.

**Comment by koen-serry** [comment]
Agreed about production, however if you're enabling debug to sort out an issue eg. it's dumping it's config for every engine you create (which seems a bit excessive). You could of course set a special entry in the logging config but the config *debugMode* seems to be the switch to toggle that dump

**Comment by mkouba** [comment]
Well, I don't think that we should only log the engine config if `debugMode` is enabled. In fact, I would expect almost the opposite - the engine config is logged when `debugMode` is enabled no matter what logging level is set. If `debugMode` is disabled the config is only logged when `DEBUG` level is set.

**Comment by koen-serry** [comment]
Yes indeed, so you can dump it even on info provided the flag is set.
